WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import SwiftUI
+import WordPressKit
+
+final class DashboardBlazeCampaignCardCell: DashboardCollectionViewCell {
+    private let frameView = BlogDashboardCardFrameView()
+    private let campaignView = DashboardBlazeCampaignView()
+
+    private lazy var contentStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 0, right: 16)
+        stackView.spacing = 4
+        stackView.isLayoutMarginsRelativeArrangement = true
+        return stackView
+    }()
+
+    private var blog: Blog?
+    private weak var presentingViewController: BlogDashboardViewController?
+
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View setup
+
+    private func setupView() {
+        contentView.addSubview(frameView)
+        frameView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.pinSubviewToAllEdges(frameView, priority: UILayoutPriority(999))
+
+        frameView.add(subview: contentStackView)
+        contentStackView.addArrangedSubview(campaignView)
+
+
+        frameView.setTitle(Strings.cardTitle)
+        frameView.onHeaderTap = { [weak self] in
+            guard let self, let presentingViewController, let blog else { return }
+            BlazeFlowCoordinator.presentBlazeCampaigns(in: presentingViewController, blog: blog)
+        }
+    }
+
+    // MARK: - BlogDashboardCardConfigurable
+
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
+        self.blog = blog
+        self.presentingViewController = viewController
+
+        let viewModel = DashboardBlazeCampaignViewModel(campaign: mockResponse.campaigns!.first!)
+        campaignView.configure(with: viewModel, blog: blog)
+    }
+}
+
+private extension DashboardBlazeCampaignCardCell {
+    enum Strings {
+        static let cardTitle = NSLocalizedString("my-sites.blazeCampaigns.title", value: "Blaze campaign", comment: "Title for the card displaying blaze campaigns.")
```

Comment:
[nit] Just to be consistent with other cards
```suggestion
        static let cardTitle = NSLocalizedString("dashboardCard.blazeCampaigns.title", value: "Blaze campaign", comment: "Title for the card displaying blaze campaigns.")
```

Reply:
Fixed